components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -121,6 +121,14 @@ public class IdPManagementDAO {
     private static final Log log = LogFactory.getLog(IdPManagementDAO.class);
 
     private static final String OPENID_IDP_ENTITY_ID = "IdPEntityId";
+    private static final String NOTIFICATION_PASSWORD_ENABLE_PROPERTY
```

Comment:
Can we move these variables to a constant class and refer every places these are used. I can see few places where we have defined these as private variables.

Reply:
Moved to IdPManagementConstants with https://github.com/wso2/carbon-identity-framework/pull/5744/commits/f5e241aeecc63692c94001e3ae24b785eb7cc55e